### PR TITLE
CTV-1746: Update url manipulation logic for the true[X] ad server call

### PR DESCRIPTION
**Pull request by episcopus**
Companion pull request to https://github.com/socialvibe/TruexAdRenderer-Roku/pull/210, transfer logic manipulating the vast_config_url over to TAR itself.

Successfully tested against locally built TAR component library.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## v1.0.3
+* Updated to v1.0.4 of the TruexAdRenderer
+* Updated url manipulation logic for true[X] ad server calls
+
 ## v1.0.2
-* Updated to v1.0.3 of the TAR-HTML5 ComponentLibrary
```
Comment (tonyr59h) [inline CHANGELOG.md:5]: 🤦‍♂

**Review by tonyr59h — APPROVED** [review]
LGTM

**Comment by imann24** [comment]
👋 hi everyone! Did you know you get pull request updates if you `Watch` a repo 😂. Nice to feel involved from afar. 

Hope all is well. Missing the team!

**Comment by tonyr59h** [comment]
Hi @imann24 !! 👋  We're playing the heck outta the NES classic you left. Hope all is well with you at Qualtrics!

**Comment by imann24** [comment]
Glad to hear! 

Qualtrics is going well. Good challenge. The only downside being no NES 😉

Let me know if you (or anyone else on the PR) care to catch up over lunch sometime.

**Pull request closed without merging**